_posts/2021-02-18-survey/survey.Rmd

```diff
@@ -0,0 +1,314 @@
+
+Truth be told, though, nothing would be more helpful here than contributions from the community. Whenever you solve even the tiniest (which is often how things appear to oneself) problem, consider creating a vignette explaining what you did. Future users will be thankful, and a growing user base means that over time, it'll be your turn to find that some things have already been solved for you!
+
+Which leads me to the last -- aggregate -- point.
+
+##### Community, community, community
+
+The remaining things I'd like to mention didn't come up quite as often (individually), but taken together, they all have something in common: They all are wishes *we* happen to have, as well!
+
+This definitely holds in the abstract -- let me cite:
+
+> "Develop more of a DL community"
+>
+> "Larger developer community and ecosystem. Rstudio has made great tools, but for applied work is has been hard to work against the momentum of working in Python."
+
+We wholeheartedly agree: And building a larger community is exactly what we're trying to do. I like the formulation "a DL community" insofar it is framework-independent. In the end, frameworks are just tools, and what counts is our ability to usefully apply those tools to problems we need to solve.
+
+Concrete wishes include
+
+-   More paper/model implementations (such as TabNet).
+
+-   Facilities for easy data reshaping and pre-processing (e.g., in order to pass data to RNNs or 1dd convnets in the expected 3-d format).
+
+-   Probabilistic programming for `torch` (analogously to TensorFlow Probability)
+
+-   A high-level library (such as fast.ai) based on `torch.`
+
+If time and resources were infinite, all of these would have become reality already.
+
+But then, instead of having everything implemented by a small group of people, isn't it much more effective (and satisfying!) to have a large and diverse group of people each contribute "theirs": the things they're interested in most, or which they feel drawn to most due to their individual backgrounds? In other words, if you're motivated to do so, please feel very encouraged to contribute to the growing community!
+
+And now, on to Spark.
+
+# Spark
+
+### Areas and applications
+
+For Spark, questions broadly paralleled those asked for DL.
+
+Overall, judging from this survey (and unsurprisingly), Spark is predominantly used in industry (n = 39). For academic staff and students, n = 4 (each). Seventeen people reported using Spark in their spare time, while 34 said they wanted to use it in the future.
```

Comment:
I'd put the staff and students together and give it 8 (_don't make me do math, even if it's just addition!_)